sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -128,6 +130,33 @@ def _create_connection(self):
                                                           "k8s_namespace": self.namespace,
                                                           "k8s_configuration": self.k8s_configuration, })))
 
+    def sudo(self,
+             cmd: str,
+             timeout: Optional[float] = None,
+             ignore_status: bool = False,
+             verbose: bool = True,
+             new_session: bool = False,
+             log_file: Optional[str] = None,
+             retry: int = 1,
+             watchers: Optional[List[StreamWatcher]] = None,
+             user: Optional[str] = 'root'
+             ) -> Result:
+        # Scylla is running as root under docker/kubernetes,
+        #   so we need this mapping to handle commands for scylla user
+        if user == 'scylla':
```

Comment:
this shouldn't be done in the module, move this logic out to the relevant function that needs it